packages/cloud/serverless.yml

```diff
@@ -0,0 +1,46 @@
+service: gcp-wowarenalogs
+
+provider:
+  name: google
+  stage: ${opt:stage}
+  runtime: nodejs14
+  region: us-central1
+  project: wowarenalogs-public-dev
+  credentials: ${self:provider.project}.json
+
+plugins:
+  - serverless-google-cloudfunctions
+
+package:
+  include:
+    - 'index.js'
+    - 'dist/**'
+
+functions:
```

Comment:
should these two be separate or would it be better to make it a single "onLogFileUploaded" handler that does both parts of the work?